### DISCUSSION TITLE
no-unused-vars issue

### DISCUSSION THREAD

**Discussion by lukasa1993** [epicweb-dev/epic-stack#885] [Q&A] [answered]
i am getting warning  `'ignored' is defined but never used  @typescript-eslint/no-unused-vars with @epic-web/config/eslint`  on `"eslint": "9.15.0"` any other way to ignore variable ?

**Comment by kentcdodds** [comment, marked as answer]
Here's how it's configured: https://github.com/epicweb-dev/config/blob/a38c8d04f3dff908dbc0f6b48dcd4377b8704a55/eslint.js#L116-L124

~~Prefix the variable with an underscore and that'll do it.~~

I'm not certain what the code looks like that you're trying to do this with so it's hard for me to help.

**Reply by lukasa1993** [reply]
thanks for reply

```
{
    files: ["**/*.{ts,tsx}"],
    rules: {
      "@typescript-eslint/no-unused-vars": [
        "error",
        {
          args: "all",
          argsIgnorePattern: "^(_.*|e)$",
          caughtErrors: "all",
          caughtErrorsIgnorePattern: "^(_.*|e)$",
          destructuredArrayIgnorePattern: "^(_.*|e)$",
          varsIgnorePattern: "^(_.*|e)$",
          ignoreRestSiblings: true,
        },
      ],
    },
  },
```

this was my fix

**Reply by kentcdodds** [reply]
Why the `|e`?

**Reply by lukasa1993** [reply]
noticed we use many catch (e) {} so i just added it :)

**Reply by kentcdodds** [reply]
If you don't use the variable there you can just not provide it:

```
try {
} catch (e) {
}
```

is the same as:

```
try {
} catch {
}
```
